pgmq.go

```diff
@@ -4,18 +4,19 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"log"
 	"time"
 
-	"github.com/craigpastro/retrier"
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgconn"
 	"github.com/jackc/pgx/v5/pgxpool"
 )
 
 const vtDefault = 30
 
-var ErrNoRows = errors.New("pgmq: no rows in result set")
+var (
+	ErrNoRows = errors.New("pgmq: no rows in result set")
+	ErrPing   = errors.New("failed to ping db")
```

Comment:
Maybe for consistency?
```suggestion
	ErrPing   = errors.New("pgmq: failed to ping db")
```